File: js/management-api-v2/PatchEnvironment.js
// DocSection: cm_api_v2_patch_environment
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
// Using ES6 syntax
import { ManagementClient } from '@kentico/kontent-management';

const client = new ManagementClient({
    projectId: '<YOUR_PROJECT_ID>',
    apiKey: '<YOUR_API_KEY>'
});

const response = await client.modifyEnvironment()
    .withData(
        [
            {
                op: "rename_environment",
                value: "My Little Production"
            }
        ]
    )
    .toPromise();
// EndDocSection

File: net/management-api-v2/PatchEnvironment.cs
// DocSection: cm_api_v2_patch_environment
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Management;
using Kontent.Ai.Management.Models.Environments.Patch;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "<YOUR_API_KEY>",
    ProjectId = "<YOUR_PROJECT_ID>"
});

var response = await client.ModifyEnvironmentAsync(new[]
{
        new EnvironmentRenamePatchModel
        {
            Value = "My Little Production"
        }
});
// EndDocSection